Dockerfile

```diff
@@ -8,6 +8,9 @@ FROM python:3.6
 RUN  apt-get update -y && \
      apt-get upgrade -y
 
+# issue with latest pip version
+RUN pip install --upgrade pip~=20.2.0
```

Comment:
I don't think this is super relevant, but if you have a link to where pip is tracking the issue maybe add it here so that we can check if this comes up again in the future.